release-prereqs.md

```diff
@@ -0,0 +1,10 @@
+# Prerequisites for performing a release
+
+- access to the official CentOS CI fedora-coreos namespace
+- access to the AWS S3 fcos-builds bucket
```

Comment:
I think I have everything else in this list other than `access to the AWS S3 fcos-builds bucket`. How do I request access to that?

Reply:
We can discuss the procedure for doing this at some point and add it to this doc, though the list here should shrink as we automate more things. Anyway, will reach out to you offline for getting you those creds.